Procfile

```diff
@@ -1,4 +1,4 @@
 web: inveniomanage runserver
 cache: redis-server
-worker: celery worker --purge -E -A invenio.celery.celery --loglevel=DEBUG --workdir=$VIRTUAL_ENV
+worker: celery worker -E -A invenio.celery.celery --loglevel=INFO --workdir=$VIRTUAL_ENV
```

Comment:
This change you can probably remove from this commit as it is not relevant for this task.